source/internal/_copyNameProperty.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copies the `name` property of a function to another.
+ * @param {Function} fna Function providing the name to be set onto `fnb`.
+ * @param {Function} fnb Function that will receive the `name` of `fna`.
+ * @returns {Function} `fnb` after defining its new `name` property.
+ */
+export default function _copyNameProperty(fna, fnb) {
+  return Object.defineProperty(fnb, 'name', { value: fna.name });
```

Comment:
`name` property of a function `name` is configurable, `defineProperty` defaults `configurable` to `false`, so it must be defined

```suggestion
  return Object.defineProperty(fnb, 'name', { configurable: true, value: fna.name });
```

See
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty

Reply:
Great catch. Amending.